src/tests/ftest/daos_test/suite.yaml

```diff
@@ -191,3 +191,5 @@ daos_tests:
     test_daos_extend_simple: 5
     test_daos_rebuild_ec: 43
     test_daos_degraded_ec: 29
+  crt_timeout:
+    test_daos_oid_allocator: 60
```

Comment:
Interestingly, the default crt_timeout is 60 I believe. But our framework overrides and uses a different default

Reply:
Right, the Python test framework set its own default value for `crt_timeout` to 10 s, in order to speed up testing.